tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1107,7 +1107,8 @@ TEST (nnstreamer_capi_switch, dummy_01)
   status = ml_pipeline_destroy (handle);
   EXPECT_EQ (status, ML_ERROR_NONE);
 
-  EXPECT_EQ (*count_sink, 3U);
+  EXPECT_GT (*count_sink, 0U);
+  EXPECT_LE (*count_sink, 3U);
```

Comment:
I'd recommend to add a loop (```while (*count_sink < 3U)```) along with usleep() and timeout-counter.

Reply:
Thank you! Updated.